web/sites/default/settings.php

```diff
@@ -42,4 +42,4 @@
 /**
  * Allow all configuration to be changed.
  */
-$settings['config_readonly_whitelist_patterns'] = ['*'];
\ No newline at end of file
+$settings['config_readonly_whitelist_patterns'] = ['*'];
```

Comment:
Can we remove the settings.php update here related to newline at end of file, to help avoid potential collisions when others pull the update down?